runs/2019/pyladies-praha-jaro-ntk/info.yml

```
@@ -0,0 +1,128 @@
+derives: pyladies
+vars:
+  pyladies: true
+  coach-present: true
+plan:
+- base: install
+  date: 2019-03-07
+  materials:
+    - title: Úvodní prezentace
+      url: http://pyladies.cz/v1/s001-install/uvod-praha.html
+    - lesson: beginners/cmdline
+      type: lesson
+    - lesson: beginners/install
+      type: lesson
+    - lesson: beginners/venv-setup
+      type: lesson
+    - lesson: beginners/first-steps
+      type: lesson
+    - lesson: beginners/install-editor
+      type: lesson
+    - title: Domácí projekty
+      url: https://projekty.pyladies.cz/
+      type: homework
+    - title: Tahák na klávesnici (PDF)
+      url: https://pyvec.github.io/cheatsheets/keyboard/keyboard-cs.pdf
+      type: cheatsheet
+    - title: Tahák na příkazovou řádku a první kroky v Pythonu (PDF)
+      url: https://s3-eu-west-1.amazonaws.com/messa-shared-files/2017/09/pyladies_2_lekce_tahak.pdf
+      type: cheatsheet
+- base: hello
+  date: 2019-03-14
+  materials:
+  - lesson: beginners/hello-world
+    type: lesson
+  - lesson: beginners/print
+    type: lesson
+  - lesson: beginners/variables
+    type: lesson
+  - lesson: beginners/comparisons
+    type: lesson
+  - title: "*Nebo* anebo *a* (bonus)"
+    lesson: beginners/and-or
+    vars:
+      bonus: true
+- base: loops
+  date: 2019-03-21
+  materils:
+    - lesson: beginners/functions
+      type: lesson
+    - lesson: intro/turtle
+      type: lesson
+    - title: Tahák s užitečnými funkcemi
+      url: https://pyvec.github.io/cheatsheets/basic-functions/basic-functions-cs.pdf
+      type: cheatsheet
+- slug: while-functions
+  title: While & Vlastní funkce
+  date: 2019-03-28
+  materials:
+    - lesson: beginners/while
+      type: lesson
+    - lesson: beginners/def
+      type: lesson
+- slug: strings
+  title: Řetězce
+  date: 2019-04-04
+  materials:
+    - lesson: beginners/str
+      type: lesson
+    - title: Řetězcový tahák
+      url: https://pyvec.github.io/cheatsheets/strings/strings-cs.pdf
+      type: cheatsheet
+- slug: practice
+  title: Procvičování & soubory
+  date: 2019-04-11
+  materials:
+    - title: Procvičíme si vše, co jsme se naučily
+      url: null
+      type: special
+    - lesson: beginners/files
+      type: lesson
+- base: list
+  date: 2019-04-18
+- base: dict
+  date: 2019-04-25
+  materials:
+    - lesson: beginners/dict
+      type: lesson
+    - title: Slovníkový tahák
+      url: https://pyvec.github.io/cheatsheets/dicts/dicts-cs.pdf
+      type: cheatsheet
+- base: tests
+  date: 2019-05-02
+  title: Výjimky, moduly & testování
+- base: class
+  title: Opakování & Třídy
+  date: 2019-05-09
+  materials:
+    - lesson: beginners/class
+      type: lesson
+    - title: Tahák na třídy
+      url: https://pyvec.github.io/cheatsheets/classes/classes.pdf
+      type: cheatsheet
+- slug: inheritance
+  title: Třídy & Dědičnost
+  date: 2019-05-16
+  materials:
+    - lesson: beginners/inheritance
+      type: lesson
+- base: pyglet
+  date: 2019-05-22
```

Díky. V tabulce je čtvrtek, takže chyba.

```suggestion
  date: 2019-05-23
```